### ISSUE TITLE
placeholder text in input does not disappear, send bt not working in form (Google Form) in amp-iframe

### ISSUE THREAD

**Issue by Arnold85** [ampproject/amphtml#4662]
## How do we reproduce the issue?

Please post a public URL and ideally post a reduced test case. E.g. on jsbin.com that exhibits only your issue and nothing else.

1. Step 1 to reproduce
     create a new .html document
2. Step 2 to reproduce
   create or take an existing google form
3.
   load the google form in a normal <iframe>
4. type into one of the fields to see the placeholdertext disappear
5. replace the <iframe>-tag with an <amp-iframe>-tag
6. reload the website
7. start typing into one of the input-fields
8. See the palceholder text does not disappear, but is overridden
9. also the send button does not work

## What browsers are affected?
 tried latest firefox and latest chrome on windows 10 pc
All browsers? Some specific browser? What device type?

## Which AMP version is affected?
 latest
Is this a new issue? Or was it always broken? Paste your AMP version. You can find it in the browser dev tools.

**Comment by mkhatib** [comment]
@Arnold85 thanks for filing, do you have a jsbin ready to test this out?

**Comment by mkhatib** [comment]
This seems like a problem in Google Forms and unrelated to AMP. @ericlindley-g would you be able to triage?

**Comment by ericlindley-g** [comment]
Thanks @mkhatib — I've reached out to the Google Forms team, and will update this thread with news.

**Comment by Arnold85** [comment]
Thanks

ericlindley-g <notifications@github.com> schrieb am Mi., 24. Aug. 2016
17:02:

**Comment by ericlindley-g** [comment]
@Arnold85  We've tested on our side, and are able to get Google Forms to work inside of `amp-iframe` as long as the `sandbox` attribute on `amp-iframe` is set to, at minimum, `allow-scripts allow-same-origin allow-forms`. You can see the [`amp-iframe` documentation](https://github.com/ampproject/amphtml/blob/master/extensions/amp-iframe/amp-iframe.md#sandbox) for more info. 

Does this fix all of the issues you found?

**Comment by ericlindley-g** [comment]
Closing this as I think the issues are solved, but please re-open if that's not the case—thanks!
